Code/GraphMol/Bond.h

```diff
@@ -275,10 +277,40 @@ class Bond : public RDProps {
   BondDir getBondDir() const { return static_cast<BondDir>(d_dirTag); };
 
   //! sets our stereo code
-  void setStereo(BondStereo what) { d_stereo = what; };
+  /*!
+      STEREONONE, STEREOANY, STEREOE and STEREOZ can be set without
+      neighboring atoms specified in getStereoAtoms since they are
+      defined by the topology of the molecular graph. In order to set
+      STEREOCIS or STEREOTRANS the neighboring atoms must be set first
+      (using setStereoBonds()) to know what atoms are being considered.
+
+      <b>Notes:</b>
+        - MolOps::findPotentialStereoBonds can be used to set
+          getStereoAtoms before setting CIS/TRANS
+  */
+  void setStereo(BondStereo what) {
+    PRECONDITION(what <= STEREOE || getStereoAtoms().size() == 2,
+                 "Stereo atoms should be specified before specifying CIS/TRANS "
+                 "bond stereochemistry")
+    d_stereo = what;
+  };
   //! returns our stereo code
   BondStereo getStereo() const { return static_cast<BondStereo>(d_stereo); };
 
+  //! sets the atoms to be considered as reference points for bond stereo
+  /*!
+      These do not necessarily need to be the highest 'ranking' atoms
```

Comment:
They don't really have to be neighbors, they could be just separated by a rigid structure?